RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -218,9 +218,9 @@ boolean LX_islandAccess()
 	{
 		abort("Dude, we got Dinghy Plans... we should not be here....");
 	}
-	while((item_amount($item[Shore Inc. Ship Trip Scrip]) < 3) && (my_meat() >= 500) && (item_amount($item[Dinghy Plans]) == 0))
+	while(item_amount($item[Shore Inc. Ship Trip Scrip]) < 3 &&  item_amount($item[Dinghy Plans]) == 0)

```

Comment:
meat amount is checking in LX_doVacation() and it will return false if you do not have enough meat. Meat amount varies on certain paths which doVacation accounts for and this did not.